### PR TITLE
chore(demo): improve icon API page with shape elements

### LINKED ISSUE #9803: 🤖 - Improve API page on for icon
### Description

https://taiga-ui.dev/components/icon/API?background=@tui.external-link

let's put the filled versions of the icons in the background, because now the exact same set is transmitted both to the icon and to the background?

### PR DESCRIPTION
**Pull request by splincode**
Fixes #9803

### CODE REVIEW

File: projects/demo/src/modules/components/icon/index.html [hunk 1]
```diff
@@ -48,15 +48,31 @@
...
-            />
+            >
+                Icon name. Default
+                <code>&#64;tui.[name]</code>
+                pattern associate a name with an icon by URL via the default icon resolver. You can also simply pass a
+                direct URL path to your icon or set up own icon name patterns (
+                <a
+                    fragment="resolver"
+                    tuiLink
+                    [routerLink]="['/components/icon']"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/icon/index.html:59]: Use `DemoRoute`

File: projects/demo/src/modules/components/icon/index.ts [hunk 2]
```diff
@@ -17,12 +16,24 @@ export default class Page {
...
+    protected readonly used = Object.values(inject(TUI_PREVIEW_ICONS)); // compatibility with proprietary icons
 
     protected readonly iconVariants = [
-        '',
-        ...TUI_USED_ICONS,
+        '@tui.info',
+        '@tui.heart',
+        ...this.used,
         'https://raw.githubusercontent.com/MarsiBarsi/readme-icons/main/github.svg',
         "\"data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none' stroke='currentColor' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'><polygon points='1 6 1 22 8 18 16 22 23 18 23 2 16 6 8 2 1 6'/><line x1='8' y1='2' x2='8' y2='18'/><line x1='16' y1='6' x2='16' y2='22'/></svg>\"",
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/icon/index.ts:26]: Could you also fix this datalist, please ?
Снимок экрана 2024-11-26 в 10 08 11

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/icon/index.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/icon/index.html]: fixed

File: projects/demo/src/modules/components/icon/index.ts (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/icon/index.ts:28]: https://github.com/taiga-family/taiga-ui/issues/9842

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**